scipy/constants/_codata.py

```diff
@@ -1194,6 +1194,7 @@
 Boltzmann constant in eV/K                                  8.617 333 262... e-5     (exact)                  eV K^-1
 Boltzmann constant in Hz/K                                  2.083 661 912... e10     (exact)                  Hz K^-1
 Boltzmann constant in inverse meter per kelvin              69.503 480 04...         (exact)                  m^-1 K^-1
+characteristic impedance of vacuum                          376.730 313 668          0.000 000 057            ohm
```

Comment:
I see the numbers also match this, which basically cites NIST: https://en.wikipedia.org/wiki/Impedance_of_free_space

One question I have, as a non-expert reading through that article, is about this statement:

> ... with the [redefinition of the SI base units](https://en.wikipedia.org/wiki/2019_redefinition_of_the_SI_base_units) that came into force on 20 May 2019, the impedance of free space is subject to experimental measurement because only the speed of light in vacuum c0 retains an exactly defined value.

Isn't that pretty much the exact reason you'd have the separation described in the deleted block below?

Reply:
I think (but I'm just guessing) the main reason for the deleted block below was the fact that this one constant was missing from the 2018 collection of constants whereas it was present in every other year. So we needed another way to get it.
One other remarks: Currently our measurement and its uncertainty do **NOT** have the same number of significant figures. This is definitely not good. 
```
>>> import scipy
>>> scipy.__version__
'1.5.4'
>>> import scipy.constants
>>> scipy.constants.physical_constants["characteristic impedance of vacuum"]
(376.73031366686166, 'ohm', 5.61366546036269e-08)
```